### ISSUE TITLE
Separate the study form view into edit & view

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#617]
# TL;DR
Separate the study edit form into a view for editing and a view for seeing the details without being able to edit.

# Narrative
The current dual use of the study form is confusing to maintain permissions for (some people can see details but not make changes). This impacts situations like studies in the Demo lab, where all researchers should be able to see all details, but while they can preview the study, they have to clone it to see the details because they can't see the edit view.

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Separate study detail edit & detail views
- [ ] Buttons for edit & view added to the main study detail page per permissions
- [ ] Tests of permissions are broken out everywhere the study edit form is currently tested
- [ ] "Shared preview" option on study still only grants ability for any researcher to preview study - not READ_STUDY_DETAILS equivalent
